eclipse-scout-core/src/desktop/outline/pages/PageModel.ts

```diff
@@ -1,22 +1,30 @@
 /*
- * Copyright (c) 2010, 2023 BSI Business Systems Integration AG
+ * Copyright (c) 2010, 2025 BSI Business Systems Integration AG
  *
  * This program and the accompanying materials are made
  * available under the terms of the Eclipse Public License 2.0
  * which is available at https://www.eclipse.org/legal/epl-2.0/
  *
  * SPDX-License-Identifier: EPL-2.0
  */
-import {Form, ObjectOrChildModel, ObjectOrModel, Outline, Page, Table, TreeNodeModel} from '../../../index';
+import {Form, ObjectModelWithUuid, ObjectOrChildModel, ObjectOrModel, ObjectOrType, Outline, Page, PageDetailMenuContributor, PageParamDo, Table, TreeNode, TreeNodeModel} from '../../../index';
 
-export interface PageModel extends TreeNodeModel {
+export interface PageModel extends TreeNodeModel, ObjectModelWithUuid<TreeNode> {
+  /**
+   * The {@link PageParamDo} containing all parameters that are required to create the page.
```

Comment:
```suggestion
   * The {@link PageParamDo} containing all parameters that are required to load the page.
```
(Creating the page instance requires more data, e.g. the parent)